standard/variables.md

```diff
@@ -160,6 +160,7 @@ The default value of a variable depends on the type of the variable and is deter
 
 - For a variable of a *value_type*, the default value is the same as the value computed by the *value_type*’s default constructor ([§8.3.3](types.md#833-default-constructors)).
 - For a variable of a *reference_type*, the default value is `null`.
+- In an unsafe context, for a variable of a *pointer_type*, the default value is `null`.
```

Comment:
Is "In an unsafe context" needed here as pointer types only occur in unsafe contexts? If its not this and preceding line become:

> - For a variable of a *reference_type* or *pointer_type*, the default value is `null`.

Reply:
And potentially this should move to the unsafe section of the spec anyway?

Reply:
I'll be creating a new issue to say "let's move all pointer stuff to section 22" - and then a PR to do this.